conceptnet5/vectors/__init__.py

```diff
@@ -1,13 +1,15 @@
-from conceptnet5.nodes import standardized_concept_uri, uri_to_label
-from conceptnet5.language.lemmatize import lemmatize_uri
-from sklearn.preprocessing import normalize
 import re
-import pandas as pd
+
 import numpy as np
+import pandas as pd
+from sklearn.preprocessing import normalize
 
+from conceptnet5.languages import ALL_LANGUAGES
+from conceptnet5.nodes import standardized_concept_uri, uri_to_label
 
 DOUBLE_DIGIT_RE = re.compile(r'[0-9][0-9]')
 DIGIT_RE = re.compile(r'[0-9]')
+CONCEPT_RE = re.compile(r'/c/({})/.+'.format('|'.join(ALL_LANGUAGES)))
```

Comment:
This works, but I wonder if it would be more effective and efficient to just use `[a-z]{2,3}` here.

Maybe `[-A-Za-z]{2,}` if we want to future-proof it for the possibility that we separate `zh-Hant` from `zh-Hans`.